lib/game/gesture_detector.dart

```diff
@@ -0,0 +1,641 @@
+import 'package:flutter/widgets.dart';
+import 'package:flutter/gestures.dart';
+import 'package:flutter/foundation.dart';
+
+// From https://github.com/flutter/flutter/blob/e6b34c2b5c/packages/flutter/lib/src/widgets/gesture_detector.dart
```

Comment:
If it is already in flutter, do we need to keep it locally too?

Reply:
Yeah, the flutter GestureDetector Widget don't include the mutitouch detectores, that is why I copy and pasted it, to add then, like I said on the PR description, I intend to make a PR on Flutter eventually, removing the need for this.

Also, if we conclude that this approach is to weird, there is also my first attempt on this commit https://github.com/flame-engine/flame/commit/5d9f1cc4ac4474d98399ad262addfa04e1489237

I don't mind using this approach instead of the one currently on the PR

Reply:
I actually prefer the first approach. It's simpler, less code, it's easy to understand and doesn't copy flutter's src. if the flutter code changes before your PR is accepting we will have to keep remembering to update this file

composing two detectors, if it works, seems very flutter-esque with widgets being composed.

Reply:
I will revert to my first approach.